### DISCUSSION TITLE
Radeon HD 5450 with Catalina

### DISCUSSION THREAD

**Discussion by mickey35us** [chris1111/Legacy-Video-patch#20] [Q&A]
Hi Chris,
Thank you for all the work and assistance to everyone. I stumbled over your tool, and I think that it is neat. Tried to get it to work with a ATI radeon hd5450 in Catalina and have a quick question. After making the app and installed all the modded kexts, what settings should be required on the config.plist and the EFI folder ? I think the following is correct but just want to confirm with you.
1. not using Whatevergreen.kext in the kext folder
2. in config.plist (graphics tab) to have "Inject ATI" checked
3. In config.plist (devices tab) to have 0x68E1002 in the ATI box for device and vendor ids

Anything else that should be done to make this work ? Please advise and thanks

**Comment by chris1111** [comment]
Nothing more but as I said is not fully QE/CI like high Sierra

**Reply by mickey35us** [reply]
thanks for your reply. what about the geforce 210 card ? would one get the full QE/CI in Catalina ? and what should be put in the clover config and whether whatevergreen kext or its web driver should be used ? thanks for your guide.

**Comment by chris1111** [comment]
Yes I told this for Geforce 210
inject Nvidia no watevergreen see config.plist

**Reply by mickey35us** [reply]
thank you
